eventually/abstractmodel/models.py

```diff
@@ -0,0 +1,82 @@
+from django.db import models
+from utils.utils import LOGGER
+
+class Abstract(models.Model):
```

Comment:
maybe it is better to call this class like `AbstractModel`? It is more meaningly =)

Reply:
done